### PR TITLE
Testing pep0440 version.

### PR DESCRIPTION
**Pull request by ocefpaf**
Hopefully this will fix https://github.com/conda-forge/basemap-data-hires-feedstock/pull/2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,9 @@ env:
     - CONDA_NPY=19  CONDA_PY=34  
     - CONDA_NPY=110  CONDA_PY=35  
     - CONDA_NPY=19  CONDA_PY=35  
+  global:
```
Comment (pelson) [inline .travis.yml]: What happened here? Any idea how come this was missing?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package:
     name: basemap
-    version: 1.0.8.dev
+    version: 1.0.8.dev0
```
Comment (ocefpaf) [inline recipe/meta.yaml]: If I got PEP 0440 correctly this should not be needed, but conda still installs basemap 1.0.7 in some situations. Not sure if that is a bug in the version solver or something else.

Conda failed to find `basemap 1.0.8.dev` in Windows+Py3k and now on `Travis-CI`.

File: .travis.yml (hunk 1 above)
Comment (ocefpaf) [inline .travis.yml]: Nope. But it was not there: https://github.com/conda-forge/basemap-feedstock/blob/2290f22eefe7c04abd361ca309bd82d08af950c3/appveyor.yml

File: recipe/meta.yaml (hunk 2 above)
Comment (ocefpaf) [inline recipe/meta.yaml]: But I loved that `conda smithy rerender` got that error! I did not notice it and the binaries never made into the channel. (Which explains the Travis-CI error we got during the copy.)

**Pull request merged**